CMakeLists.txt

```diff
@@ -354,6 +354,19 @@ ELSE()
   SET(ASPECT_WITH_PERPLEX OFF)
 ENDIF()
 
+execute_process(COMMAND ls ${CMAKE_SOURCE_DIR}/contrib/WorldBuilder/VERSION RESULT_VARIABLE result OUTPUT_QUIET ERROR_QUIET)
```

Comment:
Please capitalize all of the cmake functions as in the other parts of this file (EXECUTE_PROCESS, MESSAGE, FILE, ENDIF).